src/utils/types.ts

```diff
@@ -108,6 +108,13 @@ export type UserConfig = Partial<{
    * @default true
    */
   commentOnReleasedPullRequests: boolean;
+
+  /**
+   * Set the newly created release as latest.
+   * Note: This only has an effect for GitHub releases.
+   * @default true
+   */
+  isLatestRelease: boolean;
```

Comment:
```suggestion
  useLatestRelease: boolean;
```

Is a static setting sufficient here or would it be helpful to have a hook instead to be able to use the context to decide on?

Reply:
Hmm good question. For us, doing it static on the release branch would be sufficient... for now anyway. However, implementing it as hook maybe makes sense as it's more future proof. Do you agree? Then I can adjust it.

Reply:
```suggestion
   useLatestRelease: (ctx: HookContext) => PromiseOrValue<boolean | void>;
```

I guess it doesn't hurt to have the hook, maybe someone wants to automatically set latest for `patch` versions and not for `minor` and `major` and otherwise you could easily use `useLatestRelease: () => true`.

Reply:
Apart from that pretty cool, what you've achieved with https://github.com/opencloud-eu/woodpecker-ci-config-service 😉

Reply:
Okay, I adjusted it.

> Apart from that pretty impressive, what you've achieved with https://github.com/opencloud-eu/woodpecker-ci-config-service 😉

Thanks for the kudos! I was not involved in that, but I'll make sure the right persons receive this 🙂